it/uk/gov/hmrc/minorentityidentificationfrontend/connectors/RegistrationConnectorISpec.scala

```diff
@@ -44,13 +83,25 @@ class RegistrationConnectorISpec extends ComponentSpecHelper with RegisterStub {
 
     "return RegistrationFailed" when {
       "the registration has not been successful" in {
-        stubRegister(testSautr, testRegime)(INTERNAL_SERVER_ERROR, RegistrationFailed)
+        stubRegisterUA(testCtutr, testRegime)(INTERNAL_SERVER_ERROR, RegistrationFailed)
 
-        val result = await(registrationConnector.register(testSautr, testRegime))
+        val result = await(registrationConnector.registerUA(testCtutr, testRegime))
 
         result mustBe RegistrationFailed
       }
     }
+
+    "return InternalServerException" when {
+      "the registration has not been called" in {
```

Comment:
Throws InternalServerException when{
 the registration http status is different from Ok and INTERNAL_SERVER_ERROR

Reply:
done